Automattic/jetpack
# Jetpack components: Add ThemeProvider stories for typographies and colors

This PR Adds ThemeProvider stories for typographies and colors. Although the current story is useful, I think the instances feel kind of polluted with other elements. The goal here is trying to isolate the instances as much as possible.

Typographies | Colors
-----|------
<img width="1062" alt="image" src="https://user-images.githubusercontent.com/77539/159783160-4075a1b0-bd94-4a56-9ce7-607afda2aeb9.png"> | <img width="1062" alt="image" src="https://user-images.githubusercontent.com/77539/159783206-0cd7b082-c958-4937-812d-2d34d996238a.png">


Fixes #

#### Changes proposed in this Pull Request:

* Jetpack components: Add ThemeProvider stories for typographies and colors

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Use storybook
* Go to Typographies and Colors
* Confirm you see the instances
* Confirm you can edit the text instance for Typographies story

https://user-images.githubusercontent.com/77539/159784041-9068b095-8188-421b-8980-1b12f807ffe6.mov
